addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -870,6 +851,17 @@ function addToRes(res, data) {
     }
 }
 
+let dataDeferred;
+
+export function suspendMailDataLoad() {
+    dataDeferred = new Deferred();
+
+    return function resolveLoad() {
+        dataDeferred.resolve();
+        dataDeferred = null;
```

Comment:
Do we need to reset it?
If for example there are 2 following calls, and we want to resolve them independently, this could get in the way.